clang-tools-extra/test/clang-tidy/checkers/Inputs/kokkos/Kokkos_Core_mock.h

```diff
@@ -0,0 +1,100 @@
+#pragma once
+
+//
+// Standard library stuff that shows up in Kokkos
+//
+namespace std {
+struct string {
+  string(const char *);
+  ~string();
```

Comment:
Why do you need to bother with the destructor?

Reply:
I copy pasted this from a different check, we can remove it.  The mocking is specifically not finalized, I am just trying to get it working and will expand it as needed.